configs/stage/roles/inventory.json

```
@@ -70,6 +70,35 @@
           "permission": "inventory:groups:read"
         }
       ]
+    },
+    {
+      "name": "Account Staleness and Culling Administrator",
+      "description": "Be able to read and edit Account Staleness and Culling data.",
+      "system": true,
+      "platform_default": false,
+      "admin_default": true,
+      "version": 3,
+      "access": [
+        {
+          "permission": "inventory:staleness:write"
+        },
+        {
+          "permission": "inventory:staleness:read"
+        }
+      ]
+    },
+    {
+      "name": "Account Staleness and Culling Viewer",
+      "description": "Be able to read Account Staleness and Culling data.",
+      "system": true,
+      "platform_default": false,
```

I think this should be `true`, as peer this statement:

`Regular users will have the “Inventory staleness and culling viewer” role by default`

See: https://consoledot.pages.redhat.com/docs/dev/services/rbac.html#_define_platform_default_roles
```suggestion
      "platform_default": true,
```